zoo-css-challenge/index.html

```diff
@@ -1,136 +1,220 @@
+            meters in length and weighing up to 670 pounds!
+          </li>
+          <li>
+            Tigers are easily recognizable with their dark vertical stripes and
+            reddish/orange fur.
+          </li>
+          <li>The Bengal tiger is the most common tiger.</li>
+          <li>Tigers live between 20-26 years in the wild.</li>
+          <li>
+            Unlike most other cats, tigers are great swimmers and actually like
+            the water.
+          </li>
+        </ul>
+      </section>
+
+      <section class="giraffe">
+        <h2>Giraffe</h2>
+
+        <p>
+          Giraffes are the world's tallest mammals, thanks to their towering
+          legs and long necks. A giraffe's legs alone are taller than many
+          humans—about 6 feet . These long legs allow giraffes to run as fast as
+          35 miles an hour over short distances and cruise comfortably at 10
+          miles an hour over longer distances.
+        </p>
+
+        <h3>Behavior</h3>
+
+        <p>
+          Typically, these fascinating animals roam the open grasslands in small
+          groups of about half a dozen.
+        </p>
+
+        <p>
+          Bulls sometimes battle one another by butting their long necks and
+          heads. Such contests aren't usually dangerous and end when one animal
+          submits and walks away.
+        </p>
+
+        <h3>Height and Size</h3>
+
+        <p>
+          Giraffes use their height to good advantage and browse on leaves and
+          buds in treetops that few other animals can reach (acacias are a
+          favorite). Even the giraffe's tongue is long! The 21-inch tongue helps
+          them pluck tasty morsels from branches. Giraffes eat most of the time
+          and, like cows, regurgitate food and chew it as cud. A giraffe eats
+          hundreds of pounds of leaves each week and must travel miles to find
+          enough food.
+        </p>
+      </section>
+
+      <section class="my-animal">
+        <h1>My favorite animal</h1>
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Officia
+          dolorum quibusdam atque ipsum magni accusantium perferendis at sunt!
+        </p>
+
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Officia
+          dolorem iusto dolore, similique optio sit natus pariatur nisi maiores
+          fugit.
+        </p>
+
+        <h2>Lorem ipsum dolor sit.</h2>
+
+        <p>
+          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Animi,
+          saepe! Maiores laboriosam illum molestiae. Asperiores laboriosam dicta
+          perspiciatis, ut maxime assumenda delectus facilis accusamus, dolorum
+          laborum labore quia. Neque vero ut minima eaque voluptatibus.
+        </p>
+      </section>
+
+      <section class="news">
+        <h2>News</h2>
+        <div class="card-time">
+          <div class="card">
```

Comment:
You have repeated "Which big cat are you" 3 times rather than just adding it once as an extra card

Reply:
Thanks, I've replaced it with original cards, and add only one "Which big cat are you".